static/app/components/events/interfaces/performance/spanCountChart.tsx

```diff
@@ -0,0 +1,157 @@
+import {useState} from 'react';
+
+import {BarChart} from 'sentry/components/charts/barChart';
+import ErrorPanel from 'sentry/components/charts/errorPanel';
+import LoadingPanel from 'sentry/components/charts/loadingPanel';
+import {IconWarning} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {DateString, TagValue} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import SpanCountHistogramQuery from 'sentry/utils/performance/histogram/spanCountHistogramQuery';
+import {HistogramData} from 'sentry/utils/performance/histogram/types';
+import {formatHistogramData} from 'sentry/utils/performance/histogram/utils';
+import theme from 'sentry/utils/theme';
+
+export function SpanCountChart({issue, event, location, organization}: any) {
+  const transactionNameTag = event.tags.find(tag => tag.key === 'transaction');
+  const transactionName = transactionNameTag ? transactionNameTag.value : '';
+
+  const spanHashTag = event.tags.find(
+    (tag: TagValue) => tag.key === 'performance_issue.extra_spans'
+  ) || {key: '', value: ''};
+
+  const allEventsQuery = `event.type:transaction transaction:${transactionName}`;
+  const affectedEventsQuery = `${allEventsQuery} ${spanHashTag.key}:${spanHashTag.value}`;
+
+  const [now] = useState<DateString>(new Date());
+
+  const start = issue.firstSeen;
+  const end = now?.toString();
+  const environment = [];
+  const project = [1];
+  const spanOp = event.contexts.performance_issue.op;
+
+  function renderChart(data: HistogramData, _data: HistogramData) {
+    const xAxis = {
+      type: 'category' as const,
+      truncate: true,
+      axisTick: {
+        interval: 5,
+        alignWithLabel: true,
+      },
+    };
+
+    const colors = [theme.charts.previousPeriod, ...theme.charts.getColorPalette(4)];
+    const tooltip = {
+      formatter(series) {
+        const seriesData = Array.isArray(series) ? series : [series];
+        let contents: string[] = [];
```

Comment:
```suggestion
        const contents: string[] = [];
```

Reply:
oh we're writing to contents later so it's mutable here

Reply:
Ah my bad I missed that, thought there was only data being pushed into it